fastify.d.ts

```diff
@@ -369,6 +369,11 @@ declare namespace fastify {
      * Create a shared schema
      */
     addSchema(schema: object): FastifyInstance<HttpServer, HttpRequest, HttpResponse>
+
+    /**
+     * Prints the representation of the internal radix tree used by the router
+     */
+    printRoutes(): String
```

Comment:
This should use `string` with a lowercase `s`.
Reference: https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html